components/news/_id/OtherNews.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-row>
+    <v-row
+      justify="space-between"
+    >
+      <v-col
+        cols="auto"
+      >
+        <h2 class="">
```

Comment:
Remove the empty `class` attribute
```suggestion
        <h2>
```